cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -78,6 +78,11 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
       }
 
       p("Available privileges for a particular user can be seen using `SHOW USER name PRIVILEGES`.")
+      note {
+        p("Please note that if you are using non native auth providers like LDAP, `SHOW USER PRIVILEGES` will only work in the following limited capacity:" +
+          "It is only possible for a user to get their own privileges. Other users privileges can not be listed when using a non native auth provider.")
```

Comment:
Some minor english-isms, and also change it from "you" to "the user" to be consistant.
```suggestion
        p("Please note that if a non-native auth provider like LDAP is in use, `SHOW USER PRIVILEGES` will only work in the following limited capacity:" +
          "It is only possible for a user to show their own privileges. Other users' privileges cannot be listed when using a non-native auth provider.")
```